### PR TITLE
Add missing label to "back" button

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
fix(files): add missing label to "back" button

- Fixes https://github.com/nextcloud/photos/issues/2239

No visual changes. No errors in AXEDevtools.

### CODE REVIEW

File: src/components/HeaderNavigation.vue [hunk 1]
```diff
@@ -25,6 +25,7 @@
 		<!-- Back navigation button -->
 		<NcButton v-if="!isRoot"
 			class="photos-navigation__back"
+			:aria-label="t('photos', 'Back')"
```
Comment (szaimen) [inline src/components/HeaderNavigation.vue:28]: Maybe go back?

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

**Review by ShGKme — APPROVED** [review]

**Review by susnux — APPROVED** [review]

**Review by emoral435 — APPROVED** [review]

**Pull request merged**